src/vitePreprocessor.ts

```diff
@@ -0,0 +1,100 @@
+import path from 'path'
```

Comment:
This file totally unchanged, just moved out of `index.ts` (it is still exported as the default there)